sys/xtimer/xtimer_core.c

```diff
@@ -40,13 +40,11 @@ static volatile int _in_handler = 0;
 volatile uint64_t _xtimer_current_time = 0;
 
 static xtimer_t *timer_list_head = NULL;
-static xtimer_t *long_list_head = NULL;
```

Comment:
does not need to manage a separate long_list. One timer list is actually enough.